src/components/CardForm.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import {Component, CSSProperties} from 'react';
+import {Modal, ModalHeader, ModalBody, ModalFooter, Button} from 'jsc-react-ui';
+import {Note} from './../models/Note';
+
+const styles = {
+  modal: {
+    width: 500,
+    height: 400,
+    backgroundColor: '#fff',
+    display: 'flex',
+    flexFlow: 'column',
+  },
+  input: {
+    border: 'none',
+    fontWeight: 600,
+    fontSize: '1.1em',
+    flex: 1,
+  } as CSSProperties,
+  textarea: {
+    border: 'none',
+    resize: 'none',
+    flex: 1,
+  },
+};
+
+interface Props {
+  open: boolean;
+  onSave?: (note: Note) => void;
+  onClose?: () => void;
+}
+
+export class CardForm extends Component<Props, Note> {
+
+  props: Props = {
```

Comment:
It's not a good idea mutate props. If you want set default props, you can write something like

```typescript
static defaultProps: Props = {
  someProp: 'value',
};
```

_Pro tip: React docs recommends to always set a default value for a optional prop to prevent erros in runtime. In this case, `onSave` and `onClose`._

```typescript
static defaultProps: Props = {
  onSave: () => { return; },
  // ...
};